docs-md/tutorials/embedded-connect.md

```diff
@@ -136,15 +136,18 @@ services:
 2. Get the JDBC connector
 -------------------------
 
-[Download the JDBC connector](https://www.confluent.io/hub/confluentinc/kafka-connect-jdbc)
-to your local working directory. Next, unzip the downloaded archive:
+The easiest way to download connectors for use in ksqlDB with embedded {{ site.kconnect }}
+is via the [Confluent Hub Client](https://docs.confluent.io/current/connect/managing/confluent-hub/client.html)
+included in the `ksqlDB-server` Docker image.
 
+To download the JDBC connector, use the following command:
 ```bash
-unzip confluentinc-kafka-connect-jdbc-*.zip
+docker run -v /path/to/local/dir:/share/confluent-hub-components confluentinc/ksqldb-server:0.7.0 confluent-hub install --no-prompt confluentinc/kafka-connect-jdbc:5.4.1
```

Comment:
`5.4.1` is the current latest version of the JDBC Connector, but how should we keep this version up to date as newer versions are released?

Reply:
Updated the version to `{{ site.cprelease }}` as discussed offline.